shell/src/main/java/alluxio/cli/fs/command/DuCommand.java

```diff
@@ -68,6 +79,14 @@
           .desc("display the aggregate summary of file lengths being displayed")
           .build();
 
+  private static final Option GROUP_BY_WORKER_OPTION =
+          Option.builder(GROUP_BY_WORKER_OPTION_NAME)
```

Comment:
wrong indentation, please refer to https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#javadoc-style